### ISSUE TITLE
File uploading status not changing after uploading completed

### ISSUE THREAD

**Issue by Sarun1001** [parse-community/parse-dashboard#2274]
### New Issue Checklist

- [x ] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x ] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x ] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x ] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

https://i.ibb.co/JHcYx8D/Screenshot-at-Sept-18-02-15-34.png

- File uploading status not changing after uploading completed, 

When user updates a file in a field it indefinitely shows "uploading..." status but we can see the uploaded file in s3, it should change to  "completed" or filename after finishing uploading operation

- Change label from "uploading..." to filename or "upload pending" or "upload waiting confirmation" or something more convenient before pressing add button

Currently when users select a file from device it's showing "uploading..." label in the selected view  but actually, files are not getting uploaded at that point, file uploading only starts when the user press add button, this is a confusing implementation users waits for this 'uploading...' label to change to something like 'completed', it's a bad UX 

### Steps to reproduce

- Update a file in dashboard

- Select a file to upload in dashboard

### Actual Outcome

"Uploading..." status inside view keeps displaying even after the upload is completed
Showing "Uploading..." before pressing add button

### Expected Outcome

- "Uploading..." status should change to "complete" or filename when uploading operation finishes

- Show "Uploading..." only after pressing add button, before that provide appropriate label

### Environment

Dashboard
- Parse Dashboard version: `4.1.4`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Chrome`
- Browser version: `105.0.5195.125 (Official Build) (x86_64)`

Server
- Parse Server version: `5.2.5`
- Operating system: `Ubuntu 22.04.1 LTS`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `remote-AWS`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `5.0.12`
- Node version `v18.1.0`
- FileAdapter - s3 (aws)
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `AWS`

### Logs

No logs or error messages on the browser

**Comment by patelmilanun** [comment]
i found the exact issue but not sure if the solution I have is correct or not

```
if (this.props.value !== prevProps.value) {
  this.renderCellContent();
  this.props.value._previousSave
    ?.then(res => this.renderCellContent())
    ?.catch(err => console.log(err))
}
```

we need to add this to `parse-dashboard\src\components\BrowserCell\BrowserCell.react.js` inside componentDidUpdate

@mtrezza can u help!

**Comment by mtrezza** [comment]
You could just open a PR and others (including yourself) can test if that works with the automatic dev deployment in the PR.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0-alpha.7](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0-alpha.7)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0)
